schema/tables/santa_allowed.yml

```diff
@@ -0,0 +1,28 @@
+name: santa_allowed
+description: List of applications Santa has allowed to run on the host.
+platforms: 
+  - darwin
+evented: false
+examples: |-
```

Comment:
@allenhouchins @harrisonravazzolo when either of you get the chance, can you please help me add example queries for each of the Santa tables? I think we could start with the queries y'all have been dogfooding / demoing.

Also, please feel free to add `notes`. I omitted them here.

These examples and notes show up on fleetdm.com (website) and the Fleet product:

<img width="913" height="281" alt="Screenshot 2025-07-28 at 1 16 32 PM" src="https://github.com/user-attachments/assets/dc77a37b-ed27-43be-a3c8-e5e657ae4c31" />

<img width="292" height="406" alt="Screenshot 2025-07-28 at 1 16 49 PM" src="https://github.com/user-attachments/assets/e4d17902-fd71-4d46-a2b2-2d4cab691d5f" />

Reply:
Thanks Harry!